test/core/keyTests.ts

```diff
@@ -0,0 +1,45 @@
+﻿///<reference path="../testReference.ts" />
+
+describe("DatasetKey", () => {
+  it("is passed to Dataset", () => {
+    let ds: Plottable.Dataset = new Plottable.Dataset();
+    let key = (d: any, i: number) => { return i; };
+    ds.keyFunction(key);
+    assert.deepEqual(ds.keyFunction(), key, "key is passed to dataset");
```

Comment:
Please use `assert.strictEqual` here.  `deepEqual` is for arrays and objects I believe.